neo/SmartContract/InteropService.NEO.cs

```diff
@@ -18,34 +18,45 @@ namespace Neo.SmartContract
-        public static readonly uint Neo_Contract_Update = Register("Neo.Contract.Update", Contract_Update);
-        public static readonly uint Neo_Contract_GetScript = Register("Neo.Contract.GetScript", Contract_GetScript, 1);
-        public static readonly uint Neo_Contract_IsPayable = Register("Neo.Contract.IsPayable", Contract_IsPayable, 1);
-        public static readonly uint Neo_Storage_Find = Register("Neo.Storage.Find", Storage_Find, 1);
-        public static readonly uint Neo_Enumerator_Create = Register("Neo.Enumerator.Create", Enumerator_Create, 1);
-        public static readonly uint Neo_Enumerator_Next = Register("Neo.Enumerator.Next", Enumerator_Next, 1);
-        public static readonly uint Neo_Enumerator_Value = Register("Neo.Enumerator.Value", Enumerator_Value, 1);
-        public static readonly uint Neo_Enumerator_Concat = Register("Neo.Enumerator.Concat", Enumerator_Concat, 1);
-        public static readonly uint Neo_Iterator_Create = Register("Neo.Iterator.Create", Iterator_Create, 1);
-        public static readonly uint Neo_Iterator_Key = Register("Neo.Iterator.Key", Iterator_Key, 1);
-        public static readonly uint Neo_Iterator_Keys = Register("Neo.Iterator.Keys", Iterator_Keys, 1);
-        public static readonly uint Neo_Iterator_Values = Register("Neo.Iterator.Values", Iterator_Values, 1);
-        public static readonly uint Neo_Iterator_Concat = Register("Neo.Iterator.Concat", Iterator_Concat, 1);
+        public static readonly uint Neo_Crypto_CheckSig = Register("Neo.Crypto.CheckSig", Crypto_CheckSig, 0_01000000);
+        public static readonly uint Neo_Crypto_CheckMultiSig = Register("Neo.Crypto.CheckMultiSig", Crypto_CheckMultiSig, GetCheckMultiSigPrice);
+        public static readonly uint Neo_Header_GetVersion = Register("Neo.Header.GetVersion", Header_GetVersion, 0_00000400);
+        public static readonly uint Neo_Header_GetMerkleRoot = Register("Neo.Header.GetMerkleRoot", Header_GetMerkleRoot, 0_00000400);
+        public static readonly uint Neo_Header_GetNextConsensus = Register("Neo.Header.GetNextConsensus", Header_GetNextConsensus, 0_00000400);
+        public static readonly uint Neo_Transaction_GetWitnesses = Register("Neo.Transaction.GetWitnesses", Transaction_GetWitnesses, 0_00010000);
+        public static readonly uint Neo_Transaction_GetScript = Register("Neo.Transaction.GetScript", Transaction_GetScript, 0_00000400);
+        public static readonly uint Neo_Witness_GetVerificationScript = Register("Neo.Witness.GetVerificationScript", Witness_GetVerificationScript, 0_00000400);
+        public static readonly uint Neo_Account_IsStandard = Register("Neo.Account.IsStandard", Account_IsStandard, 0_00030000);
+        public static readonly uint Neo_Contract_Create = Register("Neo.Contract.Create", Contract_Create, 100_00000000);
+        public static readonly uint Neo_Contract_Update = Register("Neo.Contract.Update", Contract_Update, 10_00000000);
```

Comment:
`Create` and `Update` will be fixed? not related to the script size or features?

Reply:
Good point! I will modify it.